### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User did not agree to telemetry"

### DISCUSSION THREAD

**Discussion by GitScorz** [community/community#13767] [Copilot Conversations]
How I can agree to telemetry? Didn't see anything..

**Comment by GitScorz** [comment]
Nvm solved..

**Reply by rhodaax** [reply]
how did you solve?

**Reply by EndersonF** [reply]
Uninstall the GitHub Copilot extension close Vscode and then install it again, and then a message will appear for you to accept telemetry. @rhodaax 
I solved it this way!

**Reply by 1664234934** [reply]
怎么解决的？

**Reply by Ivan-G0nzalez** [reply]
It worked!
